mozilla/fx-private-relay
# remove old intro pricing news entry

This PR removes the intro pricing news entry: the only part of the code that dependended on `fast-check`, so we don't have to keep up with its updates.

How to test:
1. `cd frontend && npm run watch`
2. `python manage.py collectstatic`
3. `python manage.py runserver`
4. Sign in to http://127.0.0.1:8000/accounts/profile/
5. Click "News" in the top navigation
6. Click the "History" pill button
   * [x] You should NOT see the news item about introductory pricing (note: a waffle flag might already be hiding it)
   * [x] The rest of the News items should all still work
